### PR TITLE
Adding fitbenchmarking.com css style

### LINKED ISSUE #384: "back" hyperlinks
The generated html pages link to children but not back up the parents.
For example if you navigate from an index page to a table, there is no link back to the index page.

A link on each page to go up a level would fix this.

### LINKED ISSUE #414: Consistent style sheet
The style sheet in `HTML_templates` is different to that of `fitbenchmarking.com`.

### PR DESCRIPTION
**Pull request by wathen**
#### Description of Work

Fixes #414 
Fixes #384 

Pulls out style sheet from `fitbenchmarking.com` and adds back buttons to output.

#### Testing Instructions

1. Check the style of the results are the same as `fitbenchmarking.com`.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

File: fitbenchmarking/HTML_templates/table.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+table {
+  border-collapse: collapse;
+}
+table, th, td {
+  border: 1px solid black;
+  height: min-content;
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/HTML_templates/table.css]: I played around a bit more and found that setting the following fixes the cell height issue:
```
table, th, td {
  border: 1px solid black;
  width: auto;
  overflow: hidden;
}
th a, td a {
  display: block;
  text-decoration: none;
  color: rgb(30, 30, 30);
  margin-top: -10em;
  margin-bottom: -10em;
  padding-top: 10em;
  padding-bottom: 10em;
}
```

File: fitbenchmarking/HTML_templates/table.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+th a, td a {
+  display:block;
+  text-decoration:none;
+  color: rgb(30, 30, 30);
+  height: 100%;
+  width: 100%;
+}
+
+body {
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/HTML_templates/table.css]: This should be moved to the other css file

**Comment by tyronerees** [comment]
Some comments (for discussion) from me:
- on the second level index page: Fitbenchmarking -> FitBenchmarking
- Can the index page be styled like the 'posts' page on the website https://fitbenchmarking.com/posts/ (i.e., left justified, same fonts, slightly more padding)
- related, it's looking locally for for Fira Sans locally at `fitbenchmarking/fonts/FiraSans/ttf/FiraSans-Regular.ttf` (and `-bold.ttf`), but can't find it
- I'd rather text above/below the tables is left-justified -- I find it uncomfortable to read sentences of centered text.

File: fitbenchmarking/HTML_templates/problem_index_page.html [hunk 3]
```diff
@@ -1,32 +1,46 @@
...
+                        <form>
+                            <button class="btn default" type="submit" formaction="{{ link }}">
+                            <i class="fa fa-table"></i>
+                            </button>
+                        </form>
+                        </h3>
+                    {% endfor %}
+                </div>
+                <center>
+                <div class="centre">
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/HTML_templates/problem_index_page.html]: This is redundant

**Pull request merged**